rig.js

```diff
@@ -100,28 +104,32 @@ class RigManager {
   }
 
   async setPeerAvatarUrl(url, peerId) {
-    await this.peerRigQueue.lock();
-
-    const o = await new Promise((accept, reject) => {
-      new GLTFLoader().load(url, accept, xhr => {}, reject);
-    });
-    o.scene.traverse(o => {
-      if (o.isMesh) {
-        o.frustumCulled = false;
-      }
-    });
-    let peerRig = this.peerRigs.get(peerId);
-    this.scene.remove(peerRig.model);
-    peerRig = new Avatar(o, {
-      fingers: true,
-      hair: true,
-      visemes: true,
-      // decapitate: selectedTool === 'firstperson',
-    });
-    this.scene.add(peerRig.model);
-    this.peerRigs.set(peerId, peerRig);
-
-    await this.peerRigQueue.unlock();
+    try {
```

Comment:
Why try? If this is meant to catch fetch errors, it should be scoped tighter.

Reply:
URL can bounce and was crashing site, uncaught promise rejection